Plots/MapProjections/NCL_native_2.py

```diff
@@ -29,13 +27,13 @@
 import cartopy.feature as cfeature
 
 import geocat.datafiles as gdf
-from geocat.viz import util as gvutil
+
 ###############################################################################
 # Read in data:
 # Open a netCDF data file using xarray default engine and
```

Comment:
In sphinx, since "Read in data:" and the comment after it don't have a line between them they will both show up as headers. I'm not sure if that's what you want, but I suggest running `make html` before merging just to be sure.